src/lib/__tests__/remark-plugin-adjust-link-urls.test.ts

```diff
@@ -2,21 +2,71 @@ import { preAdjustUrl } from 'lib/remark-plugin-adjust-link-urls'
 
 describe('remarkPluginAdjustLinkUrls', () => {
 	describe('preAdjustUrl', () => {
-		describe('does not pre-adjust urls that start with: `.`, `/`, `?`, `#`', () => {
+		describe('does not pre-adjust query strings or anchors', () => {
 			const mockCurrentPath = '/some/docs/path'
 			const urlsToTest = [
-				'../oneLevelUp',
-				'./differentPageAtCurrentLevel',
-				'/root/level/page',
-				'?paramKey=paramValue',
-				'#page-anchor',
+				'.',
+				'.invalid/folder/relative',
+				'?param=value',
+				'#heading',
+				'?param=value#heading',
 			]
-
 			test.each(urlsToTest)('"%s" is not pre-adjusted', (url: string) => {
 				expect(preAdjustUrl({ currentPath: mockCurrentPath, url })).toEqual(url)
 			})
 		})
 
+		describe('pre-adjusts folder-relative urls starting with `../`', () => {
```

Comment:
❓just curious: have you found many folder relative urls like this in the content so far?

Reply:
Yep! Here's a Sourcegraph search that shows some: https://hashicorp-mktg.sourcegraph.com/search?q=context:global+lang:markdown+file:%5C.mdx%24+%5D%28../&patternType=standard&sm=0